config/routes.rb

```diff
@@ -7,6 +7,9 @@
     get '/dashboard', to: "orders#index"
     resources :users, only: [:index, :show]
     get '/merchants/:id', to: "merchants#show", as: :merchant
+    get '/merchants', to: "merchants#index", as: :merchants
+    get '/merchant/:id/enable', to: "merchants#enable", as: :merchant_enable
```

Comment:
Can we make these more RESTful by making them `PATCH` calls to "merchants#update"?

Reply:
Looks like this was changed in your next PR